### DISCUSSION TITLE
Interrupt support

### DISCUSSION THREAD

**Discussion by m-kru** [esynr3z/corsair#104] [Questions]
Does Corsair support interrupts? I do not mean exactly the same way SystemRDL does, but is there any support for interrupts?

**Comment by esynr3z** [comment]
I'm not really familiar with System RDL interrupts, but I googled it a bit and it looks like this some sugar over bitfields that implements some access pattern. Answering the question, no, there is no such concept in corsair, but I almost sure that you can create such behavior using "bricks" corsair provide.
